lib/hero-image-header/index.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component, PropTypes } from 'react';
 require('./style.css');
 const signpost = require('../../src/assets/goingplaces.svg');
+const bouncingArrow = require('../../src/assets/down-arrow.svg');
```

Comment:
yeah webpack requires that you require the svg in first and then assign it to the image src because it bundles the assets folder into the font folder so if you explicitly say that you need for example an svg from ../../src/assets/cancel.svg it won't be able to resolve the file path